src/shortcut-manager.tsx

```diff
@@ -192,6 +194,8 @@ export class ShortcutManager {
         this.store.modifiers[modifier] = false;
       }
     );
+
+    installDebug && installDebug.forceUpdate();
```

Comment:
![image](https://user-images.githubusercontent.com/334149/61540617-94d32480-aa03-11e9-9935-c0160ad15842.png)

The symbols at the top right should highlight on keydown.

Reply:
Fixed